data/membership-schema/src/main/resources/net/corda/schema/membership/group/policy/1.0/corda.group.policy.json

```diff
@@ -259,6 +259,13 @@
                 "description": "A member template as a map of values for a member to be added to the static network member list.",
                 "type": "object"
               }
+            },
+            "groupParameters": {
```

Comment:
Should `corda.minimum.platform.version` be explicitly added here or is it intentionally left out?

Reply:
We were relying on the verification done during parsing, but would be good to have it in the schema. Added in [592198a](https://github.com/corda/corda-api/pull/1180/commits/592198a818280b06d34d277d8cbc25ff56cf03f4)